src/components/token/Tokens.js

```diff
@@ -19,263 +19,293 @@ import ErrorMessageWithIcon from '../error/ErrorMessageWithIcon'
  * Displays custom tokens in a table with pagination buttons and a search bar.
  */
 class Tokens extends React.Component {
+  /**
```

Comment:
Did you run some kind of formatter in this file? If so, should we make it default for the project, so everyone always use the same rules?

Reply:
I didn't, I manually re-idented it to 2 spaces as the other files have 2 space identation